src/renderer/components/ReactFlowBox.tsx

```diff
@@ -422,6 +472,45 @@ export const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlo
         []
     );
 
+    const onLayout = useCallback(() => {
+        getLayoutedElements(nodes, edges)
+            .then((response) => {
+                const { nodes: layoutedNodes } = response;
+                if (layoutedNodes) {
```

Comment:
The API of `getLayoutedElements` doesn't make sense to me. What does `nodes: undefined` even mean? If it means "error", then why not return a rejecting promise?